src/EditorFeatures/Core/Implementation/Diagnostics/AbstractAggregatedDiagnosticsTagSource.cs

```diff
@@ -52,6 +53,11 @@ protected override void Disconnect()
 
         public override ITagSpanIntervalTree<TTag> GetTagIntervalTreeForBuffer(ITextBuffer buffer)
         {
+            return GetAccurateTagIntervalTreeForBuffer(buffer, CancellationToken.None);
```

Comment:
@pharring I guess this is just to make us behave same as before? asking since this will not return accurate result at the point of asking.

we probably need an issue to return correct data in update 1?